config/src/main/java/org/springframework/security/config/annotation/web/builders/FilterComparator.java

```diff
@@ -73,6 +73,9 @@
 		filterToOrder.put(
 			"org.springframework.security.oauth2.client.web.OAuth2AuthorizationRequestRedirectFilter",
 				order.next());
+		filterToOrder.put(
+				"org.springframework.security.saml2.provider.service.servlet.filter.SamlServiceProviderMetadataFilter",
```

Comment:
I think this needs to be changed to match the changes made to `Saml2MetadataFilter`'s name and location.